setuptools/monkey.py

```diff
@@ -130,7 +130,7 @@ def patch_func(replacement, target_mod, func_name):
 
 
 def get_unpatched_function(candidate):
-    return getattr(candidate, 'unpatched')
+    return candidate.unpatched
```

Comment:
It is hard to guess what was the original intention but maybe the idea was to have a

```suggestion
    return getattr(candidate, 'unpatched', None)
```

but there was a writing mistake?